### ISSUE TITLE
Training loss 0 for LLama2 on A40

### ISSUE THREAD

**Issue by JiayongO-O** [XiongjieDai/GPU-Benchmarks-on-LLM-Inference#21]
Great work. I am wondering if you have trained the LLama2 model on A40 model? I am encountering a problem to train the LLama2 model on A40 48G. The precision is fp16, but the loss and grad_norm are 0 all the time. Don't know if you have encountered similar problem or having any suggestions. Thanks.
Detailed configs:
deepspeed==0.14.4 (tried 0.12.3, 0.15.2 all fail)
accelerate==0.31.0
peft==0.8.2
transformers==4.42.0.
